services/bettersocial/post/createPostV3.js

```diff
@@ -228,7 +228,8 @@ async function processSendSystemMessage(
               resolve();
             })
             .catch((e) => {
-              reject(e);
+              // TODO: Temporary fix, implement the sending message in queue if fails.
+              console.log('error send system message', e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While logging the error is a good first step, it's not a robust solution for error handling. It would be better to implement a retry mechanism or queue the message for later delivery when an error occurs. Also, avoid using `console.log` for logging errors in production code. Consider using a dedicated logging library that can provide more context and control over log levels.

```diff
- // TODO: Temporary fix, implement the sending message in queue if fails.
- console.log('error send system message', e);
+ // TODO: Implement the sending message in queue if fails.
+ logger.error('Error sending system message', e);
```

Please replace `logger.error` with your preferred logging method.


<!-- This is an auto-generated comment by OSS CodeRabbit -->